joint_state_controller:
  type: joint_state_controller/JointStateController
  publish_rate: 100

lwr_control:
  type: stack_of_passive_controllers_controller/StackOfPassiveControllersController
  joints:
    - lwr_a1_joint
    - lwr_a2_joint
    - lwr_e1_joint
    - lwr_a3_joint
    - lwr_a4_joint
    - lwr_a5_joint
    - lwr_a6_joint
  passive_controllers:
    lwr_3_link:
      Wmax: [10., 10., 10., 0., 0., 0.]
      Err0: [0.01, 0.01, 0.01, 0.057, 0.057, 0.057]
      Errb: [0.02, 0.02, 0.02, 0.63, 0.63, 0.63]
      K0: [10., 10., 10., 0., 0., 0.]
    lwr_6_link:
      # Wmax: [5000., 5000., 5000., 25., 25., 25.]
      # Err0: [0.01, 0.01, 0.01, 0.057, 0.057, 0.057]
      # Errb: [0.02, 0.02, 0.02, 0.63, 0.63, 0.63]
      # K0: [5000., 5000., 5000., 25., 25., 25.]
      Wmax: [180., 180., 180., 0., 0., 0.]
      Err0: [0.005, 0.005, 0.005, 0.057, 0.057, 0.057]
      Errb: [0.02, 0.02, 0.02, 0.63, 0.63, 0.63]
      K0: [180., 180., 180., 0., 0., 0.]
      Orientation: true
    # lwr_7_link:
    #   Wmax: [50., 50., 50., 0., 0., 0.]
    #   Err0: [0.01, 0.01, 0.01, 0.057, 0.057, 0.057]
    #   Errb: [0.02, 0.02, 0.02, 0.63, 0.63, 0.63]
    #   K0: [10., 10., 10., 0., 0., 0.]
    #   Orientation: true
